config/io_defaults.sh

```diff
@@ -23,6 +23,7 @@
 THIS=$(basename -- $0)
 allMessage "$THIS: Setting default parameters for input/output."
 #
+NSCREEN=-1000 # frequency (in time steps) of output to adcirc.log
```

Comment:
If we want to be able to set this in a configuration file; then I think this needs to be something like,

```
NSCREEN=${NSCREEN:-"-1000"}
```

Do you thing?

Reply:
Yes, I have now set it that way in the code in this PR.